phpcs.xml.dist

```diff
@@ -9,7 +9,7 @@
     <config name="php_version" value="70100"/>
 
      <!-- Ignore warnings and show progress of the run -->
-    <arg value="np"/>
+    <arg value="nps"/>
```

Comment:
The PR looks good. What does this option do though? I'll be AFK for a week, so I can't run `--help`

Reply:
It makes PHPCS output the identifiers of failing sniffs. We use that option on other repos as well.

Reply:
Ah yes that one is great!